package.json

```diff
@@ -54,7 +54,7 @@
         "express": "^4.16.4",
         "express-http-context": "^1.2.0",
         "express-rate-limit": "^3.5.0",
-        "ganache-core": "^2.5.6",
```

Comment:
You should revert this change. We try to keep packages the same, then upgrade them explicitly as part of an issue.

You can use `npm ci` to install the exact versions that are in the package-lock.json

Reply:
Done